src/Files.App/ViewModels/Properties/CustomizationViewModel.cs

```diff
@@ -58,16 +71,14 @@ public CustomizationViewModel(IShellPage appInstance, BaseProperties basePropert
 			else
 				return;
 
+			DllIcons = [];
```

Comment:
Can you move this to L24?

Reply:
Like this?
`public ObservableCollection<IconFileInfo> DllIcons { get; } = [];`